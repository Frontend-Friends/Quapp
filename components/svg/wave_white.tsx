import { SVGProps } from 'react'

export const WaveWhiteSVG = (props: SVGProps<SVGSVGElement>) => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 375 43.64"
      role="none"
      {...props}
    >
      <g>
        <path fill="#FFFFFF" d="M0,0V45.64H375V16.5C223-23.5,95,108,0,0Z" />
      </g>
    </svg>
  )
}
