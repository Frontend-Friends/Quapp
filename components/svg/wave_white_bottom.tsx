import { SVGProps } from 'react'

export const WaveWhiteBottomSVG = (props: SVGProps<SVGSVGElement>) => {
  return (
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 375 154.53" {...props}>
      <g>
        <path
          fill="#FFFFFF"
          d="M.56,126.08c-.18-.46-.37-.94-.56-1.43v29.87H375V0C359.09,121.63,26.54,190.26,.56,126.08Z"
        />
      </g>
    </svg>
  )
}
