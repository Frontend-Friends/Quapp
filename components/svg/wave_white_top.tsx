import { SVGProps } from 'react'

export const WaveWhiteTopSVG = (props: SVGProps<SVGSVGElement>) => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 375 63.52"
      role="none"
      {...props}
    >
      <g>
        <path
          fill="#FFFFFF"
          d="M.56,47.34C56.94-36.33,211.15,18.38,375,63.52V0H0V48.2c.19-.29,.37-.58,.56-.86Z"
        />
      </g>
    </svg>
  )
}
