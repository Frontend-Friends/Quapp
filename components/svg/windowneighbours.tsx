import { SVGProps } from 'react'
import { useTranslation } from '../../hooks/use-translation'

export const WindowNeighboursSVG = (props: SVGProps<SVGSVGElement>) => {
  const t = useTranslation()
  return (
    <svg
      width="495"
      height="236"
      viewBox="0 0 495 236"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      role="img"
      {...props}
    >
      <title id="windowNeighboursTitle">{t('SVG_windowneighbours')}</title>
      <g clipPath="url(#clip0_726_4922)">
        <path d="M233 217H-3V234H233V217Z" fill="#E6E6E6" />
        <path
          d="M167.568 14.1071H9.3831V114.184H167.568V14.1071Z"
          fill="white"
        />
        <path d="M177.57 110.079H19.3819V217.34H177.57V110.079Z" fill="white" />
        <path
          d="M92.5845 13.5939H84.367V114.184H92.5845V13.5939Z"
          fill="#3F3D56"
        />
        <path
          d="M167.568 13.5939H159.351V114.184H167.568V13.5939Z"
          fill="#3F3D56"
        />
        <path
          d="M167.568 105.972H9.38307V114.184H167.568V105.972Z"
          fill="#3F3D56"
        />
        <path
          d="M31.7383 67.9945L17.6007 77.4907V85.347L39.0178 67.9945H31.7383Z"
          fill="white"
        />
        <path
          d="M55.0317 67.9945L17.6007 97.2757V103.968L66.3872 67.9945H55.0317Z"
          fill="#E6E6E6"
        />
        <path
          d="M17.6005 13.5939H9.3831V114.184H17.6005V13.5939Z"
          fill="#3F3D56"
        />
        <path
          d="M144.711 21.8054L113.128 61.836L122.886 62.8621L150.106 20.2657L144.711 21.8054Z"
          fill="#E6E6E6"
        />
        <path
          d="M167.568 13.5939H9.3831V21.8053H167.568V13.5939Z"
          fill="#3F3D56"
        />
        <path
          d="M167.568 59.7831H9.38309V67.9945H167.568V59.7831Z"
          fill="#3F3D56"
        />
        <path
          d="M51.4143 215.808V219.082C51.4143 219.082 49.2298 236 43.2223 236C37.2148 236 42.13 218.536 42.13 218.536V215.808L51.4143 215.808Z"
          fill="#9E616A"
        />
        <path
          d="M142.619 154.139L141.527 151.41C141.527 151.41 134.427 141.041 139.342 138.313C144.258 135.584 150.265 149.227 150.265 149.227L151.357 153.593L142.619 154.139Z"
          fill="#9E616A"
        />
        <path
          d="M119.955 150.808C127.916 150.808 134.371 144.359 134.371 136.403C134.371 128.447 127.916 121.997 119.955 121.997C111.993 121.997 105.539 128.447 105.539 136.403C105.539 144.359 111.993 150.808 119.955 150.808Z"
          fill="#9E616A"
        />
        <path
          d="M109.578 138.04L99.7481 145.68L113.401 157.14L121.046 147.863L109.578 138.04Z"
          fill="#9E616A"
        />
        <path
          d="M118.549 217.34H68.3468L72.1541 176.995L74.9802 147.033V147.027C75.0407 146.384 75.2274 145.76 75.5298 145.189C75.8322 144.618 76.2442 144.113 76.7423 143.701C77.2404 143.29 77.8148 142.981 78.4326 142.792C79.0504 142.603 79.6995 142.537 80.3427 142.599L101.174 144.571L116.503 153.377L116.949 157.416L132.245 168.328L121.849 188.091L115.31 200.529L118.549 217.34Z"
          fill="#57B894"
        />
        <path
          d="M75.4444 172.694L61.2448 191.249L52.5066 217.445H39.3993L48.6836 178.697L73.1908 145.212C73.8212 144.355 74.7119 143.725 75.73 143.415C76.7481 143.105 77.8392 143.133 78.8405 143.493C79.1533 143.622 79.4797 143.715 79.8135 143.77L75.4444 172.694Z"
          fill="#57B894"
        />
        <path
          d="M120.228 187.429L154.361 201.345L160.368 195.343L151.904 151.41L140.981 153.594L145.35 179.789L132.243 168.329L127.873 167.237L120.228 187.429Z"
          fill="#57B894"
        />
        <path
          d="M105.838 141.069C105.148 135.337 105.79 129.523 107.715 124.08C110.914 115.501 121.111 118.086 126.918 120.032C132.725 121.977 140.42 130.116 138.418 134.558C136.416 139 126.914 133.388 123.525 130.527C120.135 127.666 114.214 136.355 114.214 136.355L105.838 141.069Z"
          fill="#2F2E41"
        />
        <path
          d="M116.052 135.913C116.584 134.45 116.356 133.024 115.543 132.729C114.729 132.433 113.638 133.38 113.106 134.843C112.574 136.306 112.802 137.731 113.615 138.026C114.429 138.322 115.519 137.375 116.052 135.913Z"
          fill="#9E616A"
        />
      </g>
      <path d="M433 217H286V234H433V217Z" fill="#E6E6E6" />
      <path d="M391.46 13.8159H233.134V113.914H391.46V13.8159Z" fill="white" />
      <path
        d="M420.458 109.807H262.135V217.091H420.458V109.807Z"
        fill="white"
      />
      <path
        d="M316.409 13.3026H308.185V113.914H316.409V13.3026Z"
        fill="#3F3D56"
      />
      <path
        d="M391.46 13.3026H383.235V113.914H391.46V13.3026Z"
        fill="#3F3D56"
      />
      <path d="M391.46 105.7H233.134V113.914H391.46V105.7Z" fill="#3F3D56" />
      <path
        d="M255.509 67.7146L241.359 77.2133V85.0707L262.795 67.7146H255.509Z"
        fill="#E6E6E6"
      />
      <path
        d="M278.824 67.7146L241.359 97.0019V103.696L290.189 67.7146H278.824Z"
        fill="#E6E6E6"
      />
      <path
        d="M241.359 13.3026H233.134V113.914H241.359V13.3026Z"
        fill="#3F3D56"
      />
      <path
        d="M368.583 21.5158L336.971 61.5548L346.738 62.5811L373.982 19.9758L368.583 21.5158Z"
        fill="#E6E6E6"
      />
      <path
        d="M391.46 13.3026H233.134V21.5158H391.46V13.3026Z"
        fill="#3F3D56"
      />
      <path
        d="M391.46 59.5015H233.134V67.7147H391.46V59.5015Z"
        fill="#3F3D56"
      />
      <path
        d="M337.795 134.129C337.795 134.129 338.398 136.134 334.985 137.938C331.572 139.742 330.568 146.559 334.583 148.563C338.598 150.568 339.803 153.174 337.795 154.979C335.788 156.783 332.174 161.595 336.992 162.798C341.811 164 342.413 166.406 341.008 167.409C339.602 168.411 344.22 165.604 344.22 165.604C344.22 165.604 345.023 170.416 349.239 169.213C353.455 168.01 365.3 145.155 359.277 141.947C353.254 138.74 337.795 134.129 337.795 134.129Z"
        fill="#2F2E41"
      />
      <path
        d="M332.317 179.851L313.053 193.369C313.053 193.369 298.476 178.292 298.476 169.454C298.476 160.615 290.666 143.978 287.022 150.737C283.377 157.496 291.187 173.613 291.187 173.613L305.764 207.926C305.764 207.926 314.095 209.486 318.78 205.846C323.466 202.207 336.482 194.929 336.482 194.929L332.317 179.851Z"
        fill="#FFB9B9"
      />
      <path
        d="M351.32 157.236C359.371 157.236 365.897 150.718 365.897 142.679C365.897 134.639 359.371 128.122 351.32 128.122C343.269 128.122 336.742 134.639 336.742 142.679C336.742 150.718 343.269 157.236 351.32 157.236Z"
        fill="#FFB9B9"
      />
      <path
        d="M343.77 152.817C343.77 152.817 348.977 166.334 346.894 169.454C344.812 172.573 342.209 172.573 340.126 173.613C339.085 174.133 328.151 178.292 328.151 181.411C328.202 183.416 328.738 185.378 329.713 187.13L331.275 195.968L344.291 204.287L361.472 202.207L374.488 194.929C374.488 194.929 376.57 175.692 370.843 172.573C365.116 169.454 358.348 162.175 358.869 159.575C359.389 156.976 359.91 150.737 359.91 150.737L343.77 152.817Z"
        fill="#FFB9B9"
      />
      <path
        d="M371.366 192.331V192.348L370.897 196.939L370.3 202.828L369.62 209.486L368.85 217.091H331.123C330.615 212.934 329.717 208.527 328.149 205.848C327.163 204.253 326.725 202.381 326.899 200.516C327.251 197.034 329.711 194.604 329.711 193.891C329.711 193.111 325.326 188.537 323.133 186.292C322.397 185.54 321.906 185.049 321.906 185.049L328.673 179.851L328.768 180.013L329.717 181.673L332.835 187.128L331.798 191.289C331.798 191.289 333.41 191.568 335.653 191.925C339.53 192.532 345.299 193.368 347.933 193.368C350.377 193.368 353.362 194.621 355.828 195.34C355.906 195.362 355.978 195.384 356.051 195.407C357.602 195.841 358.93 196.042 359.767 195.54C359.817 195.512 359.865 195.48 359.912 195.446C360.348 195.151 360.82 194.913 361.318 194.738H361.323C363.126 194.101 364.972 193.591 366.847 193.212C368.956 192.749 370.808 192.426 371.26 192.348C371.261 192.347 371.261 192.347 371.262 192.346C371.263 192.346 371.264 192.346 371.265 192.346C371.266 192.346 371.267 192.346 371.268 192.346C371.269 192.347 371.27 192.347 371.271 192.348C371.332 192.337 371.366 192.331 371.366 192.331Z"
        fill="#D80062"
      />
      <path
        d="M372.989 191.54C372.95 191.819 372.917 192.103 372.883 192.381C372.722 193.607 372.588 194.833 372.471 196.059C372.147 199.312 371.952 202.575 371.885 205.848C371.866 207.069 371.737 208.287 371.5 209.486C370.982 212.066 370.285 214.607 369.413 217.091H354.673C355.063 214.194 355.588 209.736 355.844 205.068V205.062C355.883 204.338 355.911 203.608 355.939 202.878C356.036 200.425 356.002 197.968 355.839 195.518C355.833 195.457 355.833 195.396 355.828 195.34C355.8 195.022 355.772 194.716 355.744 194.409C354.706 184.013 360.949 172.575 360.949 172.575C363.064 172.575 365.128 173.739 366.969 175.344C369.218 177.35 370.941 179.874 371.99 182.697C373.039 185.519 373.382 188.555 372.989 191.54Z"
        fill="#FFB9B9"
      />
      <path
        opacity="0.1"
        d="M371.885 205.848C371.865 207.069 371.737 208.287 371.5 209.486C370.982 212.066 370.285 214.607 369.413 217.091H354.561C355.015 212.407 355.155 207.698 354.979 202.995C354.952 202.304 354.913 201.602 354.857 200.906V200.894C354.818 200.293 354.767 199.691 354.706 199.089C354.614 198.394 354.652 197.687 354.818 197.006V196.994C354.936 196.477 355.22 196.011 355.627 195.669C355.693 195.612 355.764 195.562 355.839 195.518C355.905 195.473 355.976 195.436 356.051 195.407C356.119 195.369 356.19 195.338 356.263 195.312C356.845 195.12 357.464 195.066 358.07 195.156C358.648 195.225 359.216 195.354 359.766 195.54C362.225 196.44 364.522 197.73 366.568 199.362C367.928 200.386 369.179 201.547 370.3 202.828C371.282 203.987 371.885 205.056 371.885 205.848Z"
        fill="black"
      />
      <path
        d="M354.183 198.048L355.843 205.067L365.116 209.486H372.926L376.57 192.849C376.57 192.849 370.136 191.49 366.845 193.209C363.554 194.929 355.827 195.34 355.827 195.34L354.183 198.048Z"
        fill="#D80062"
      />
      <path
        d="M351.439 128.068C351.439 128.068 352.106 130.289 348.325 132.288C344.544 134.287 343.432 141.838 347.88 144.059C352.328 146.28 353.663 149.167 351.439 151.166C349.215 153.165 345.211 158.495 350.549 159.827C355.887 161.16 356.554 163.825 354.997 164.936C353.44 166.046 358.556 162.937 358.556 162.937C358.556 162.937 359.445 168.267 364.116 166.934C368.786 165.602 373.446 141.379 368.24 135.14C363.399 129.339 351.439 128.068 351.439 128.068Z"
        fill="#2F2E41"
      />
      <path
        d="M350.018 125.782C350.019 122.528 370.271 126.1 374.45 130.643C377.344 133.789 381.74 138.522 378.057 141.858C374.373 145.195 373.868 148.334 376.549 149.66C379.23 150.986 384.525 155.041 379.744 157.762C374.964 160.482 375.041 163.228 376.84 163.878C378.639 164.528 372.874 162.912 372.874 162.912C372.874 162.912 373.456 168.284 368.599 168.258C363.741 168.232 349.806 141.575 353.135 134.166C356.231 127.276 350.018 125.782 350.018 125.782Z"
        fill="#2F2E41"
      />
      <path
        d="M352.129 125.069C339.094 124.285 337.785 134.084 337.785 134.084C337.875 139.755 346.893 136.992 349.398 137.837C352.42 135.666 358.244 131.559 362.41 128.628C359.385 126.502 355.823 125.268 352.129 125.069Z"
        fill="#2F2E41"
      />
      <defs>
        <clipPath id="clip0_726_4922">
          <rect width="495" height="236" fill="white" />
        </clipPath>
      </defs>
    </svg>
  )
}
