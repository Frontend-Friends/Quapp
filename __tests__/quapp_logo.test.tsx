import { render } from '@testing-library/react'
import { LogoSVG } from '../components/svg/quapp_logo'

it('renders Quapp Logo unchanged', () => {
  const { container } = render(<LogoSVG title={'Quapp Logo'} />)
  expect(container).toMatchSnapshot()
})
