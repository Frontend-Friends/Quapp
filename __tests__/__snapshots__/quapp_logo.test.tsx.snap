// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Quapp Logo unchanged 1`] = `
<div>
  <svg
    fill="none"
    height="42"
    role="img"
    title="Quapp Logo"
    viewBox="0 0 196 66"
    width="140"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      filter="url(#filter0_d_672_120)"
    >
      <path
        d="M3.99998 20.3488C3.99998 9.03552 12.9855 3.57393 22.6736 3.57393C32.3617 3.57393 41.3473 9.03552 41.3473 20.3488C41.3473 31.6621 32.3617 37.1237 22.6736 37.1237C12.9855 37.1237 3.99998 31.6621 3.99998 20.3488ZM28.8457 20.3488C28.8457 11.1434 27.283 5.13438 22.6736 5.13438C18.0643 5.13438 16.5016 11.1434 16.5016 20.3488C16.5016 29.5542 18.0643 35.4846 22.6736 35.4846C27.283 35.4846 28.8457 29.5542 28.8457 20.3488Z"
        fill="url(#paint0_linear_672_120)"
      />
      <path
        d="M102.762 29.2428C102.349 24.2468 102.762 9.18971 102.762 9.18971C102.762 6.38027 101.59 5.13127 100.418 5.13127C99.246 5.13127 98.0739 6.38026 98.0739 8.79959C98.0739 10.5929 98.8553 12.5434 100.576 13.0122C99.4823 14.494 97.6833 15.5857 94.0128 15.5857C90.0273 15.5857 87.6832 13.1663 87.6832 10.4356C87.6832 6.53442 91.2781 3.57082 99.7942 3.57082C108.31 3.57082 115.264 6.38026 115.264 16.2117V27.2136C115.264 30.6459 116.514 32.0522 118.077 32.0522C118.468 32.0522 118.937 31.9736 119.407 31.7408C118.783 34.0028 116.124 37.1237 110.969 37.1237C107.453 37.1237 103.959 35.0693 103.077 31.1147C102.973 30.6522 102.803 29.7147 102.765 29.2428H102.762Z"
        fill="url(#paint1_linear_672_120)"
      />
      <path
        d="M180.068 19.9587C180.068 27.761 175.639 31.8194 171.722 31.8194C170.613 31.8194 169.508 31.4293 168.619 30.8064C169.801 33.6158 172.756 36.3466 177.85 36.3466C186.417 36.3466 191.883 29.0918 191.883 19.9618C191.883 10.8319 186.417 3.57707 177.85 3.57707C172.756 3.57707 169.801 6.30787 168.619 9.11732C169.505 8.49439 170.613 8.10428 171.722 8.10428C175.636 8.10428 180.068 12.1627 180.068 19.965V19.9587Z"
        fill="url(#paint2_linear_672_120)"
      />
      <path
        d="M156.804 13.2481V43.614C156.804 44.3722 156.726 45.3255 155.67 45.8006C155.689 45.8006 156.852 45.8918 156.886 45.8949C159.955 46.1435 162.403 46.5808 163.799 46.8608C166.256 47.3547 168.493 47.9682 170.497 48.6194C168.802 47.2981 168.622 45.9421 168.622 44.2779V3.57393C165.667 5.29169 159.76 6.77349 154.221 7.39641C156.804 7.70787 156.804 10.1272 156.804 13.2481Z"
        fill="url(#paint3_linear_672_120)"
      />
      <path
        d="M143.143 19.9587C143.143 27.761 138.71 31.8194 134.797 31.8194C133.688 31.8194 132.583 31.4293 131.694 30.8064C132.876 33.6158 135.831 36.3466 140.925 36.3466C149.492 36.3466 154.958 29.0918 154.958 19.9618C154.958 10.8319 149.492 3.57707 140.925 3.57707C135.831 3.57707 132.876 6.30787 131.694 9.11732C132.579 8.49439 133.688 8.10428 134.797 8.10428C138.71 8.10428 143.143 12.1627 143.143 19.965V19.9587Z"
        fill="url(#paint4_linear_672_120)"
      />
      <path
        d="M118.32 56.2801C119.577 55.3961 121.609 54.0464 124.221 52.6464C125.67 51.8693 128.153 50.6172 131.36 49.4185C131.499 49.3651 133.399 48.8302 133.477 48.8019C132.831 48.7642 131.694 48.0594 131.694 46.7758C131.694 45.3161 131.694 31.294 131.694 30.8064V3.57707C128.739 5.29483 122.831 6.77664 117.293 7.39956C119.876 7.71102 119.876 10.1303 119.876 13.2513V52.1462C119.876 53.6311 119.281 54.9934 118.32 56.2833V56.2801Z"
        fill="url(#paint5_linear_672_120)"
      />
      <path
        d="M61.2717 30.259C56.9743 30.259 55.8023 27.0594 55.8023 20.8176C55.8023 20.6603 55.8023 12.9713 55.8023 12.814C55.8023 11.3857 56.404 11.0207 56.6939 10.7407C56.7096 10.725 55.7455 10.8823 55.7014 10.8854C55.2194 10.9358 54.5672 10.9955 53.7953 11.0364C53.7953 11.0364 52.7777 11.0931 51.76 11.0931C48.7984 11.0931 45.4431 10.6023 42.68 9.95419C43.3448 10.5362 43.3006 11.7506 43.3006 13.2513V23.5515C43.3006 30.8064 45.8023 37.1268 54.3971 37.1268C60.8811 37.1268 64.4759 32.9143 65.6479 28.0756C64.6334 29.4787 62.9132 30.259 61.2717 30.259Z"
        fill="url(#paint6_linear_672_120)"
      />
      <path
        d="M78.1495 27.2136V2.35327C78.1495 1.8027 78.096 1.00674 78.6568 0C77.2642 1.61079 74.7343 3.3663 72.261 4.98653C70.1375 6.3771 67.2232 7.85575 64.8319 8.79957C65.7865 9.36272 65.6479 12.0023 65.6479 13.2481V28.4625C65.6479 34.0028 69.5547 37.1237 73.8521 37.1237C79.0096 37.1237 81.6656 34.0028 82.2894 31.7408C81.82 31.9736 81.3505 32.0522 80.9598 32.0522C79.3971 32.0522 78.1463 30.6491 78.1463 27.2136H78.1495Z"
        fill="url(#paint7_linear_672_120)"
      />
      <path
        d="M102.724 21.1007C101.955 20.7169 101.165 20.5281 100.204 20.5281C96.8105 20.5281 95.1628 23.3596 95.1628 25.5619C95.1628 27.4495 96.1079 30.281 99.8887 30.281C100.85 30.281 102.271 29.7776 103.039 29.3938C103.039 32.1686 100.519 36.8877 94.7217 37.2023C86.9807 37.6239 82.5603 33.1124 82.5603 27.1349C82.5603 21.1574 87.1067 17.0675 94.5326 17.0675C99.5736 17.0675 101.7 19.3735 102.724 21.1007Z"
        fill="url(#paint8_linear_672_120)"
      />
      <path
        d="M18.0864 31.9484C18.1777 31.8383 18.8142 31.0833 19.6207 30.8882C19.9326 30.8127 20.5911 30.7372 21.6087 31.3916C22.201 31.7722 24.1418 34.3268 25.9818 36.9381C27.6138 39.3826 29.4821 41.5974 31.5205 43.7053C38.8488 51.2873 48.7102 56.7552 59.3656 57.7274C70.2478 58.7215 79.2932 57.1044 87.6769 51.0954C91.9208 47.424 94.1168 45.5552 97.2453 41.028C89.4161 43.3687 84.1483 40.8801 81.9901 37.7718C81.7822 40.446 79.9327 42.4406 79.6082 42.7583C78.2976 44.0325 77.7399 44.4383 75.5943 45.7565C70.3454 48.9844 66.3977 49.384 60.2036 49.8433C48.2187 50.633 35.9314 44.0671 27.7807 35.5287L24.9515 31.8603C23.8047 30.517 22.7965 29.1107 20.9345 28.7394C20.6352 28.6796 18.8582 28.5223 17.3617 29.7493C17.557 30.5641 17.7965 31.2909 18.0801 31.9484H18.0864Z"
        fill="url(#paint9_linear_672_120)"
      />
    </g>
    <defs>
      <filter
        color-interpolation-filters="sRGB"
        filterUnits="userSpaceOnUse"
        height="66"
        id="filter0_d_672_120"
        width="195.883"
        x="-2.28882e-05"
        y="0"
      >
        <feflood
          flood-opacity="0"
          result="BackgroundImageFix"
        />
        <fecolormatrix
          in="SourceAlpha"
          result="hardAlpha"
          type="matrix"
          values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
        />
        <feoffset
          dy="4"
        />
        <fegaussianblur
          stdDeviation="2"
        />
        <fecomposite
          in2="hardAlpha"
          operator="out"
        />
        <fecolormatrix
          type="matrix"
          values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.15 0"
        />
        <feblend
          in2="BackgroundImageFix"
          mode="normal"
          result="effect1_dropShadow_672_120"
        />
        <feblend
          in="SourceGraphic"
          in2="effect1_dropShadow_672_120"
          mode="normal"
          result="shape"
        />
      </filter>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_672_120"
        x1="31.4922"
        x2="13.8963"
        y1="35.5978"
        y2="5.07784"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.18"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint1_linear_672_120"
        x1="121.511"
        x2="87.695"
        y1="28.45"
        y2="8.89904"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.28"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint2_linear_672_120"
        x1="186.474"
        x2="170.591"
        y1="33.4868"
        y2="5.94203"
      >
        <stop
          offset="0.09"
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.47"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint3_linear_672_120"
        x1="174.612"
        x2="152.715"
        y1="46.2473"
        y2="8.26569"
      >
        <stop
          offset="0.09"
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.47"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint4_linear_672_120"
        x1="149.545"
        x2="133.666"
        y1="33.4868"
        y2="5.94199"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.18"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint5_linear_672_120"
        x1="135.311"
        x2="113.358"
        y1="47.7448"
        y2="9.66564"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.18"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint6_linear_672_120"
        x1="59.3971"
        x2="43.8793"
        y1="36.1736"
        y2="9.26159"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.18"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint7_linear_672_120"
        x1="86.3631"
        x2="63.1851"
        y1="25.0491"
        y2="11.6462"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.35"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint8_linear_672_120"
        x1="98.7356"
        x2="88.3861"
        y1="36.1012"
        y2="18.1525"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.18"
          stop-color="white"
        />
      </lineargradient>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint9_linear_672_120"
        x1="17.368"
        x2="97.2485"
        y1="43.3529"
        y2="43.3529"
      >
        <stop
          stop-color="#C8C8C8"
        />
        <stop
          offset="0.08"
          stop-color="white"
        />
        <stop
          offset="0.77"
          stop-color="white"
        />
        <stop
          offset="1"
          stop-color="#C8C8C8"
        />
      </lineargradient>
    </defs>
  </svg>
</div>
`;
